### DISCUSSION TITLE
希望网页中可以隐藏掉API Key的值

### DISCUSSION THREAD

**Discussion by wwf523399102** [ChatGPTNextWeb/NextChat#1547] [Q&A]
考虑到域名会共享给其他用户，为了保证个人隐私和API Key被盗用，在事先填入环境变量中的API Key可以自动隐藏，不用显示。那么就可以有效地保护域名创建者所提供的API Key，避免API Key被滥用。

**Comment by Yidadaa** [comment]
API Key 不会被用户看到，网页里的输入框是让用户填自己的 key 用的，你可以自己加个环境变量隐藏掉它，具体看文档。
